lib/compat/wordpress-6.5/block-patterns.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Extends Core's wp-includes/block-patterns.php to add new media related
+ * pattern categories for WP 6.5.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Adds new pattern categories for better organization of media related patterns.
+ *
+ * Note: This should be removed when the minimum required WP version is >= 6.5.
+ *
+ * @return void
+ */
+function gutenberg_register_media_pattern_categories() {
+	register_block_pattern_category(
+		'images',
+		array(
+			'label'       => _x( 'Images', 'Block pattern category' ),
+			'description' => __( 'Different layouts containing images.' ),
```

Comment:
I'm wondering what is the difference between the new category "Images" and the existing [Gallery](https://github.com/WordPress/WordPress/blob/a6b095398c577c02802e8727b009e5a951533aaf/wp-includes/block-patterns.php#L128) because their description looks the same:

Gallery: Different layouts for displaying images.
Images: Different layouts containing images.

Reply:
Good point. I've pushed a commit that tweaks the gallery description to hopefully differentiate between it and the images category some.

My understanding from the discussion in https://github.com/WordPress/gutenberg/issues/47847 is that there is a consensus towards keeping the Gallery category alongside the new Images category.

/cc @richtabor can you confirm we do still want to keep Gallery?

Reply:
I'm thinking Gallery, Audio, and Video would cover all the bases well. Images isn't necessarily needed, and I'd think that we'd shift relevant gallery patterns to the gallery category.

Reply:
Thanks for the direction here @richtabor.

I've removed the addition of the Images category in favour of retaining the original Gallery category.

If you don't have any concerns over leaving the Media category registered (See https://github.com/WordPress/gutenberg/issues/47847#issuecomment-1756931356), I'll look to get this one merged.

Reply:
Works for me!